janusgraph-benchmark/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="warn">
+    <Appenders>
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss} %-5level %class.%method{36} - %msg%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Root level="info">
```

Comment:
The logging level needs to be at least the warning level, otherwise, there will be [too many logs](https://github.com/JanusGraph/janusgraph/runs/4703815873?check_suite_focus=true#step:6:933).